CHANGELOG.md

```diff
@@ -67,6 +67,8 @@ The following example shows how you can use the Indigo theme:
     - `activeViewChanged` event emitted after the active view (DEFAULT, YEAR, DECADE) is changed after user interaction.
     - `viewDate` day value is always 1.
     - `activeView` setter is now available as an input property.
+- `IgxCombo`
+    - Added `showCaseIcon` to display a case sensitive search icon in the search input. Icon click allows the user to easily toggle the search case sensitivity.
```

Comment:
This may have already being discussed, yet I would prefer a bit longer yet more meaningful description for the property `showCaseIcon` --> `showMatchCaseIcon` OR `showCaseSensitiveIcon`

Reply:
Renamed to showSearchCaseIcon